netket_tools/python_utils.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Netket python utilities.
+"""
+import json
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def encode_complex(z):
+    if isinstance(z, complex):
+        return (z.real, z.imag)
+    else:
+        type_name = z.__class__.__name__
+        raise TypeError(
+            f"Object of type '{type_name}' is not JSON serializable")
+
+
+def get_obsv_from_json(outputfile):
```

Comment:
Yes sorry I missed this function! But this will return all observables, not a specific one, right?

Reply:
Correct, this returns a dictionary of np.ndarrays storing the mean and other statistics for each observable. Were you looking for it to return only one?

Reply:
Yes, I think it's nice to have a function asking for a specific observable instead of all of them...